README.md

```diff
@@ -216,6 +216,44 @@ following information:
   **OPTIONAL**, if not assigned, logs will be streamed to `System.out`. When assigned, XTF will attempt to create the 
   path in case it doesn't exist and default to `System.out` should any error occur. 
 
+###### Run test cases in separate namespaces using `xtf.openshift.unique.namespace.pertestcase=true` property and in parallel
```

Comment:
This is kind of personal preference objection. First I saw this property name it looks little bit weird to me. 

Why word `per` is not delimited with dot?  So we would have something like

`xtf.openshift.unique.namespace.per.testcase`

Do we need `unique` part? Leading to something like

`xtf.openshift.namespace.per.testcase`